Caffeine/Caffeine/ViewControllers/LoginViewController.swift

```diff
@@ -70,23 +73,28 @@ extension LoginViewController {
     }
 
     @objc private func turnOnHomeScreen() {
-        if loginViewModel.checkForInvalidInut(self.emailField) == true {
-            self.showAlertButtonTapped(self.loginButton)
-            self.emailField.text = ""
-            self.passwordField.text = ""
+        if self.wrongInputPromt != nil { // Dismiss the view from here
+            self.wrongInputPromt.removeFromSuperview()
+            self.timerForPromt.invalidate()
```

Comment:
Cia meto warning'a: Comparing non-optional value of type 'UILabel' to 'nil' always returns true